icons/map-pin-check-inside.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M20 10c0 4.993-5.539 10.193-7.399 11.799a1 1 0 0 1-1.202 0C9.539 20.193 4 14.993 4 10a8 8 0 0 1 16 0" />
```

Comment:
The check feels a bit high.

Reply:
I can't decide if it looks better when moved 1px down or not:
![image](https://github.com/lucide-icons/lucide/assets/17746067/19d445d1-d6f6-4248-af61-fe64d7d9b195)

On the other hand, the centering was very deliberate:
<img width="260" alt="image" src="https://github.com/lucide-icons/lucide/assets/17746067/e91686ca-7fe6-4fea-95fa-3a097857c09a">

Reply:
feels better with it being moved 1px down to me

Reply:
@ericfennis?

Reply:
I agree with @jguddas !

Reply:
This would result in using a different centering for the checkmark than all other elements though. 🤔

Not sure if I'm a fan of that.
![image](https://github.com/user-attachments/assets/c4cbac1a-fb86-4d73-85a7-a8c9f42008f8)

Reply:
The visual center of the checkmark is different tho.
When putting those side by side, the checkmark feels like it has a higher center of gravity to me.
![image](https://github.com/user-attachments/assets/59af2f55-24a0-4c30-ba4d-889502932b7e)

Reply:
I'm fine with disregarding this tho, and keeping it with a bit of a high perceptual center.

Reply:
@karsa-mistmere Ok then we should go for the top row!